lib/feature/add_character/cubit/add_character_cubit.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:character_viewer/common/network/dto/character.dart';
+import 'package:character_viewer/common/service/character_service.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:injectable/injectable.dart';
+
+part 'add_character_cubit.freezed.dart';
+part 'add_character_state.dart';
+
+@injectable
+class AddCharacterCubit extends Cubit<AddCharacterState> {
+  final CharactersService _charactersService;
+
+  AddCharacterCubit(this._charactersService)
+      : super(const AddCharacterState.initial());
+
+  void addCharacter({
+    required String title,
```

Comment:
it's not critical but if you wanna do it better you can create Character in UI when you call addCharacter method and pass only one parameter (Character character)

Reply:
Done
Removed